api/v1alpha1/storage_webhook.go

```diff
@@ -67,13 +68,28 @@ var _ webhook.Validator = &Storage{}
 func (r *Storage) ValidateCreate() error {
 	storagelog.Info("validate create", "name", r.Name)
 
+	configuration := make(map[string]interface{})
```

Comment:
Actually, in my other PR about static creds that is currently in progress, I also needed some data from YDB yaml config. What do you think about creating a very simple type [like this](https://github.com/ydb-platform/ydb-kubernetes-operator/pull/91/files#diff-7e297919d1f39a103313fc13ba994fc39308661fc49000b88d72da69108dec00R106) instead of unmarshalling it to ungodly `map[string]interface{}`, and then writing custom logic to check for `nil`s if parsing failed?

Reply:
We need properly managed configuration schema with versioning. I suggest to fix this later after ydb configuration subsystem modernization.

Reply:
Decided to make partially typed config structures and parsing in different merge request.